test/bounded-memory/mzcompose.py

```diff
@@ -36,8 +35,7 @@
 ITERATIONS = 128
 
 SERVICES = [
-    Cockroach(setup_materialize=True),
```

Comment:
Cockroach was not needed here?

Reply:
Correct. the Materialized is always overridden and then uses an internal cockroach (now postgres). I could also fix it in the other direction (make Postgres actually external), but then this might make the test a bit lighter since there is more memory for environmentd.